### PR TITLE
added residential address indicator at address builder

### PR DESCRIPTION
**Pull request by thejackwu**

### CODE REVIEW

**Comment by thejackwu** [comment]
@caseyli Updated PR to include an empty tag, instead of a boolean.

**Review by Dujota — APPROVED** [review]
@jackwu833016 ok sweet so we can still re-use the element_with_value(TagName, Value).

File: lib/ups/builders/address_builder.rb [hunk 1]
```diff
@@ -128,6 +128,7 @@ def to_xml
           # By setting the residential address indicator tag.
           # We can trigger UPS to check for any surcharge related to that area
           # and apply the charge automically.
+          # ** WARNING: The usage of an empty flag is not inlcuded in UPS's documentation, subject to future changes.
```
Comment (Dujota) [inline lib/ups/builders/address_builder.rb]: the documentation states this is an empty tag, check page 24 of the developer guide

Comment (thejackwu) [reply lib/ups/builders/address_builder.rb]: Nice catch! Removed that warning from PR

**Pull request merged**